tests/snapshots/tests.contrib.vertexai.test_vertexai.test_vertexai_completion_async_stream_tool.json

```diff
@@ -0,0 +1,41 @@
+[[
+  {
+    "name": "vertexai.request",
+    "service": "tests.contrib.vertexai",
+    "resource": "GenerativeModel.generate_content_async",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "",
+    "error": 0,
+    "meta": {
+      "_dd.p.dm": "-0",
+      "_dd.p.tid": "6737a11600000000",
+      "language": "python",
+      "runtime-id": "0f7964e043bd4142b009292431d64054",
+      "vertexai.request.contents.0.text": "What is the weather like in New York City?",
+      "vertexai.request.generation_config.max_output_tokens": "30",
+      "vertexai.request.generation_config.stop_sequences": "['x']",
+      "vertexai.request.generation_config.temperature": "1.0",
+      "vertexai.request.model": "gemini-1.5-flash",
+      "vertexai.request.provider": "google",
+      "vertexai.request.stream": "True",
+      "vertexai.response.candidates.0.content.parts.0.function_calls.0.function_call.args": "{'location': 'New York City, NY'}",
+      "vertexai.response.candidates.0.content.parts.0.function_calls.0.function_call.name": "get_current_weather",
+      "vertexai.response.candidates.0.content.parts.0.text": "",
```

Comment:
hmm is this missing the actual return value from the function call? or is the response just blank

Reply:
The interaction from this test involves asking the model a question and getting a function call as the response. Tracing the actual response from the function call would require another interaction.